src/main/scala/org/scalajs/tools/tsimporter/Importer.scala

```diff
@@ -195,14 +196,21 @@ class Importer(val output: java.io.PrintWriter) {
     }
   }
 
-  private def processPropertyDecl(owner: ContainerSymbol, name: Name,
+  private def processPropertyDecl(moduleRoot: ContainerSymbol, owner: ContainerSymbol, name: Name,
       tpe: TypeTree, modifiers: Modifiers, protectName: Boolean = true) {
     if (name.name != "prototype") {
       tpe match {
         case ObjectType(members) if members.forall(_.isInstanceOf[CallMember]) =>
           // alternative notation for overload methods - #3
           for (CallMember(signature) <- members)
             processDefDecl(owner, name, signature, protectName)
+        case ObjectType(members) =>
+          val inner = moduleRoot.newAnonMemberName()
+          val nested = moduleRoot.getClassOrCreate(Name(inner))
+          processMembersDecls(moduleRoot, owner, nested, members)
+
```

Comment:
Don't put a blank line here, or alternatively surround the `case` itself with blank lines above and below. Otherwise the parts of the case body visually group with the other cases above and below, which is confusing.